doc/source/data/mars-on-ray.rst

```diff
@@ -7,14 +7,7 @@ Using Mars on Ray
 
 
 `Mars`_ is a tensor-based unified framework for large-scale data computation which scales Numpy, Pandas and Scikit-learn.
```

Comment:
Can you update the MARS bullet in https://docs.ray.io/en/master/data/dataset.html#supported-input-formats to be a checkmark instead of TODO?

Reply:
done